README.md

```diff
@@ -89,22 +89,35 @@ endpoint of the tax calculator in order to provide the most coherent and
 reliable results. TaxJar support recommends using this endpoint for live
 calculations.
 
-## Development
+## Development Setup
 
-Before installing dependencies, make sure your environment variables are setup.
-Our CI uses something like this:
-`SOLIDUS_BRANCH=v2.10 DB=postgresql RAILS_VERSION="~> 5.2.0"`.
+The environment variables needed for development are listed in `.env.sample`. Copy these to `.env`, replace the appropriate values, and run `source .env` in your terminal.
```

Comment:
Curious how others feel about this as a step we encourage for development on the extension. It is quite verbose to specify the full environment before each command, but at least it is explicit. I am worried this may introduce issues for people who are not used to having to source this file every time before running a command in a new shell since we don't use `dotenv` in this gem.

Reply:
Yeah, I might be more inclined to just provide a list of all of the environment variable combinations that are under CI, and then let users copy-paste those as a convenience.

I will personally just be including the environment variables in my commands.

We could always provide `dotenv` as a development gem, too.

Reply:
People should just use direnv. 😅

Reply:
I liked one suggestion @jarednorman had which is to update the defaults / fallback for these environment variables to things we consider "sane" in the `Gemfile`. The defaults we have here don't seem appropriate for the current state of the gem
https://github.com/SuperGoodSoft/solidus_taxjar/blob/master/Gemfile#L5
https://github.com/SuperGoodSoft/solidus_taxjar/blob/master/Gemfile#L12

Reply:
These instructions won't work in all shells. 🐟 

I think rather than telling people "run this command", it's fine to say set these variables (since there's only one required and one optional.) You can provide the commands to export them in the README, but using a `.env` file is confusing:

- It might lead anyone familiar with dotenv to think that file would actually be loaded by something, which isn't the case.
- The file doesn't match the format of `.env` files, but of `direnv` files, which would be named `.envrc`.

Why don't we just tell people to set the two env variables and offer an example command to do so, in case the person doesn't know how

-----

@forkata You might consider using "sensible" instead of "sane". I'd hardly consider the existing defaults "insane". 😅

Reply:
@jarednorman sorry for the bad paraphrase. As usual your choice of words is better than mine. I share the same concerns around trying to introduce a dotfile which may imply certain convention or use of a specific gem to manage the environment. If we wanted to provide a shell script for convenience that can be probably made to work independent of the shell someone is using, but even that is a bit too opinionated for me.

I think we should guide developers to what environment they need to export or specify before each command and allow them to set that in any way they wish. I also think this would not be as big of an issue if the defaults we provided actually produced a working gem, which is not the case now.